gate-core/src/main/groovy/com/netflix/spinnaker/gate/filters/FiatSessionFilter.groovy

```diff
@@ -75,7 +75,11 @@ class FiatSessionFilter implements Filter {
             session.invalidate()
             SecurityContextHolder.clearContext()
           }
+        } else {
+          log.debug("Unable to get permission from from permission evaluator for user=${user}.")
```

Comment:
```suggestion
          log.debug("Unable to get permission from permission evaluator for user=${user}.")
```

Reply:
But, is this log message really correct?  We get here if `fiatPermission != null`, so that means we did get a permission from the permission evaluator?

Reply:
Fixed. That will teach me to push commits at 2AM.

Reply:
You are right, I misread the logic. I removed it and left the other one in, since that's the one I hit when testing and the userInfoURI was unreachable.